### PR TITLE
db/edit-mode-drag-drop-enabled

### PR DESCRIPTION
**Pull request by pawellysy**
Implemented disabling drag&drop and resizer.

### CODE REVIEW

File: ts/Dashboards/EditMode/EditMode.ts [hunk 1]
```diff
@@ -299,21 +305,14 @@ class EditMode {
...
-        // If dragDrop is disabled in options, don't init it.
-        if (!(editMode.options.dragDrop &&
-                editMode.options.dragDrop.enabled !== false)
-        ) {
-            editMode.dragDrop = new DragDrop(
-                editMode,
-                editMode.options.dragDrop
-            );
-        }
+        editMode.dragDrop = new DragDrop(
```
Comment (karolkolodziej) [inline ts/Dashboards/EditMode/EditMode.ts:312]: Maybe add a comment on why this is always initialized.

**Review by karolkolodziej — CHANGES_REQUESTED** [review]
- [ ] add test

File: samples/dashboards/cypress/edit-mode-options/demo.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<div id="container">
```
Comment (karolkolodziej) [inline samples/dashboards/cypress/edit-mode-options/demo.html:1]: one line?

File: samples/dashboards/cypress/edit-mode-options/demo.mjs [hunk 3]
```diff
@@ -0,0 +1,86 @@
+
+// Bring in other forms of Highcharts
```
Comment (karolkolodziej) [inline samples/dashboards/cypress/edit-mode-options/demo.mjs:2]: Is this relevant?

**Review by karolkolodziej — APPROVED** [review]

File: samples/dashboards/cypress/edit-mode-options/demo.mjs [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+                series: [
+                    {
+                        name: 'Series from options',
+                        data: [1, 2, 1, 4]
+                    }
+                ],
+                chart: {
+                    animation: false,
+                    type: 'pie'
+                }
```
Comment (sebastianbochan) [inline samples/dashboards/cypress/edit-mode-options/demo.mjs:53]: You can simplify chart to 

```
chartOptions: {
                series: [
                    {
                        data: [1, 2, 1, 4]
                    }
                ],
```

**Review by sebastianbochan — CHANGES_REQUESTED** [review]

File: ts/Dashboards/EditMode/Toolbar/CellEditToolbar.ts [hunk 5]
```diff
@@ -47,63 +47,55 @@ class CellEditToolbar extends EditToolbar {
...
 
                         if (dragDrop && cellEditToolbar.cell) {
                             dragDrop.onDragStart(e, cellEditToolbar.cell);
                         }
                     }
                 }
-            },
-            settings: {
+            }] :
+            [];
```
[the remark this answers is not in the document]
Comment (pawellysy) [reply ts/Dashboards/EditMode/Toolbar/CellEditToolbar.ts:75]: I'd leave it as it is :)

**Comment by sebastianbochan** [comment]
Please check the cypress test.

**Review by bre1470 — APPROVED** [review]

**Review by sebastianbochan — APPROVED** [review]

**Pull request merged**